bin/browserify-ftw.js

```
@@ -71,9 +74,7 @@ try {
 }
 
 try {
-  options = require(fullConfig);
-  // Poor man's validation
-  if (!options.quote || !options.style) throw new Error(fullConfig + 'is not a valid refactor config');
```

Why not? It makes sure to fail early on an invalid config.